hr_timesheet_sheet/models/account_analytic_line.py

```diff
@@ -38,15 +42,27 @@ def create(self, values):
     def write(self, values):
         self._check_state_on_write(values)
         res = super(AccountAnalyticLine, self).write(values)
-        vals_do_compute = ['date', 'employee_id', 'project_id', 'company_id']
+        vals_do_compute = [
+            'date', 'employee_id', 'project_id', 'task_id', 'company_id',
```

Comment:
It's necessary because you may change the task of a timesheet without changing its project, and thus this kind of change should trigger also `_compute_sheet` method

Reply:
Why you want to trigger `_compute_sheet` if you only change the task? The `sheet_id` stays the same in that case.

Reply:
🤔 yeah, but the same I could say for `project_id`, right?

Reply:
No. The project can belong to a different employee or different company.

Reply:
:thinking: oh, ok :+1: